nodestream/databases/neo4j/index_query_builder.py

```diff
@@ -10,28 +10,28 @@
 REL_FIELD_INDEX_QUERY_FORMAT = "CREATE INDEX {constraint_name} IF NOT EXISTS FOR ()-[r:`{type}`]-() ON (r.`{field}`)"
 
 
-def key_index_from_format(key_index: KeyIndex, format: str) -> Query:
+def key_index_from_format(key_index: KeyIndex, format: str, apoc_iterate: bool) -> Query:
     key_pattern = ",".join(f"n.`{p}`" for p in sorted(key_index.identity_keys))
     constraint_name = f"{key_index.type}_node_key"
     statement = format.format(
         constraint_name=constraint_name,
         key_pattern=key_pattern,
         type=key_index.type,
     )
-    return Query.from_statement(statement)
+    return Query.from_statement(statement, apoc_iterate)
 
 
 class Neo4jIndexQueryBuilder:
     """Creates index creation queries that will work for any Neo4j Database Supported."""
 
-    def create_key_index_query(self, key_index: KeyIndex) -> Query:
+    def create_key_index_query(self, key_index: KeyIndex, apoc_iterate: bool) -> Query:
```

Comment:
I don't think we should have to change the `index_query_builder` module since it doesn't do anything with batching.